### PR TITLE
Fix/classic editor design amends

### PR DESCRIPTION
**Pull request by dpanta94**
Amend design for fees appearance in tickets while in classic editor.

image
image
image

### CODE REVIEW

File: src/Tickets/Commerce/Order_Modifiers/Admin/Order_Modifier_Fee_Metabox.php [hunk 1]
```diff
@@ -236,12 +237,32 @@ public function add_fee_section( int $post_id, ?int $ticket_id, string $ticket_t
...
+	/**
+	 * Formats the fees' amount for display based on TicketsCommerce Currency Settings.
+	 *
+	 * @since TBD
+	 *
+	 * @param object $fee The fee object.
+	 *
+	 * @return object The formatted fee object.
+	 */
+	public function format_fee_for_display( Object $fee ): Object {
```
Comment (Camwyn) [inline src/Tickets/Commerce/Order_Modifiers/Admin/Order_Modifier_Fee_Metabox.php]:
```suggestion
	public function format_fee_for_display( object $fee ): object {
```

File: src/resources/postcss/tickets-admin.pcss [hunk 2]
```diff
@@ -828,27 +828,25 @@ p.description {
...
+		strong {
+			display: inline-block;
+			margin-bottom: var(--tec-spacer-1);
 		}
-	}
 
-	.selectable-fees {
-		width: 100%;
+		.tribe-dropdown .select2-selection--multiple .select2-selection__rendered {
+			border: none;
```
Comment (Camwyn) [inline src/resources/postcss/tickets-admin.pcss:844]: We typically do `border: 0`
```suggestion
			border: 0;
```

File: src/admin-views/order_modifiers/classic-fee-edit.php [hunk 3]
```diff
@@ -26,36 +26,43 @@
...
+						<select
+							class="tribe-dropdown"
+							name="ticket_order_modifier_fees[]"
+							id="ticket_order_modifier_fees"
+							multiple
+							data-placeholder="<?php esc_attr_e( 'Select fees...', 'event-tickets' ); ?>"
+							data-search-placeholder="<?php esc_attr_e( 'Search fees...', 'event-tickets' ); ?>"
+							data-allow-clear="true"
+							data-dropdown-css-width="false"
+							style="width: 100%;"
```
Comment (Camwyn) [inline src/admin-views/order_modifiers/classic-fee-edit.php:54]: We had to override this with an inline style?

**Review by Camwyn — COMMENTED** [review]
One small change and a question

File: src/admin-views/order_modifiers/classic-fee-edit.php (hunk 3 above)
Comment (dpanta94) [reply src/admin-views/order_modifiers/classic-fee-edit.php:54]: Unfortunately yes, thats how we do it in most of our codebase from what i found

File: src/Tickets/Commerce/Order_Modifiers/Admin/Order_Modifier_Fee_Metabox.php (hunk 1 above)
Comment (dpanta94) [reply src/Tickets/Commerce/Order_Modifiers/Admin/Order_Modifier_Fee_Metabox.php]: its done already

File: src/admin-views/order_modifiers/classic-fee-edit.php (hunk 3 above)
Comment (Camwyn) [reply src/admin-views/order_modifiers/classic-fee-edit.php:54]: ugh, ok

**Review by Camwyn — APPROVED** [review]

**Pull request merged**